src/applications/disability-benefits/all-claims/tests/data/transformed-data/service-info.json

```diff
@@ -0,0 +1,49 @@
+{
+  "form526": {
+    "isVaEmployee": false,
+    "homelessOrAtRisk": "no",
+    "phoneAndEmail": {
+      "primaryPhone": "4445551212",
+      "emailAddress": "test2@test1.net"
+    },
+    "mailingAddress": {
+      "country": "USA",
+      "addressLine1": "123 Main",
+      "city": "Bigcity",
+      "state": "AK",
+      "zipCode": "12345"
+    },
+    "ratedDisabilities": [
+      {
+        "name": "Diabetes mellitus0",
+        "ratedDisabilityId": "0",
+        "ratingDecisionId": "63655",
+        "diagnosticCode": 5238,
+        "decisionCode": "SVCCONNCTED",
+        "decisionText": "Service Connected",
+        "ratingPercentage": 100,
+        "disabilityActionType": "INCREASE"
+      },
+      {
+        "name": "Diabetes mellitus1",
+        "ratedDisabilityId": "1",
+        "ratingDecisionId": "63655",
+        "diagnosticCode": 5238,
+        "decisionCode": "SVCCONNCTED",
+        "decisionText": "Service Connected",
+        "ratingPercentage": 100,
+        "disabilityActionType": "NONE"
+      }
+    ],
+    "serviceInformation": {
+      "servicePeriods": [
+        {
+          "serviceBranch": "Air Force",
+          "dateRange": { "from": "2001-03-21", "to": "2014-07-21" }
+        }
+      ]
+    },
```

Comment:
This is good, but I might recommend checking for the removal of the data in a unit test instead. Like this, it'll run the puppeteer form app tester and plugin all the data, which should work perfectly, but is relatively slow.

Reply:
Oh, I didn't know these were also run on e2e tests. I've just updated this PR.

Reply:
Also? How did you expect them to be run? I might be misunderstanding something.

Reply:
The submit transformer unit test loads in each json file & transformed json file, then does a comparison - see https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/all-claims/tests/submit-transformer.unit.spec.js#L30-L56